.github/workflows/daily.yml

```diff
@@ -143,6 +143,31 @@ jobs:
         if: ${{ failure() }}
       - uses: ./.github/actions/build-teardown
 
+  prover-inconsistency-test:
+    runs-on: ubuntu-latest-xl
+    container:
+      image: diem/build_environment:main
+    env:
+      MESSAGE_PAYLOAD_FILE: /tmp/message
+    strategy:
+      fail-fast: false
+      matrix:
+        release-branch: [release-1.1]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.release-branch }}
+      - uses: ./.github/actions/build-setup
+      - name: Run Prover test with inconsistency check
+        run: |
+          MVP_TEST_INCONSISTENCY=1 cargo test -p move-prover --release
+      - uses: ./.github/actions/slack-file
+        with:
+          webhook: ${{ secrets.WEBHOOK_BREAKING_CHANGE }}
```

Comment:
I don't think this is the webhook you want to use...

Reply:
Perhaps, should I create a new one to make it report errors (if any) to the `move-prover` channel? Could anyone advise how to do that?

Reply:
you can now use:  secrets.WEBHOOK_MOVE_LANG